python/oneflow/nn/modules/random_ops.py

```diff
@@ -462,6 +463,81 @@ def randperm_op(
         ).to(dtype)
 
 
+def normal_op(
+    mean,
+    std,
+    *size: Union[_size_any_t, flow.Size, List[int]],
+    out=None,
+    placement: flow.placement = None,
+    sbp: flow._oneflow_internal.sbp.sbp = None,
+    generator=None,
+    dtype: Optional[flow.dtype] = None,
+    device: Union[flow.device, str, None] = None,
+    requires_grad: bool = False
+):
+    r"""
+    Returns a tensor of random numbers drawn from separate normal distributions whose mean and standard deviation are given.
+
+    Args:
+        mean (float):  the mean for all distributions
+        std (float):  the standard deviation for all distributions
+        size (int...):  a sequence of integers defining the shape of the output tensor.
+
+    Keyword args:
+        out (Tensor, optional):  the output tensor.
+        placement (flow.placement, optional): The desired device of returned global tensor. If None, will
+          construct local tensor.
+        sbp (flow.sbp, optional): The desired sbp of returned global tensor. It must be equal with the
+          numbers of placement.
+        generator(:class:`oneflow.Generator`, optional):  a pseudorandom number generator for sampling
+        dtype (:class:`oneflow.dtype`, optional): the desired data type of returned tensor.
+            Default: `oneflow.float32`.
+        device: the desired device of returned tensor. Default: cpu.
+        requires_grad(bool, optional): If autograd should record operations on the returned tensor. Default: False.
+
+    Example:
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> generator = flow.Generator()
+        >>> generator.manual_seed(0)
+        >>> y = flow.normal(0, 1, 5, generator=generator)
+        >>> y
+        tensor([2.2122, 1.1631, 0.7740, 0.4838, 1.0434], dtype=oneflow.float32)
+    """
+    if len(size) == 1:
+        new_shape = size[0]
+        if isinstance(new_shape, int):
+            new_shape = (new_shape,)
+    else:
+        new_shape = size
```

Comment:
不要让 new_shape 和 size 这两个名字相似作用却大不一样的变量同时存在，很容易误用，改成这样：
```suggestion
    if len(size) == 1:
        size = size[0]
        if isinstance(size, int):
            size = (size,)
```